README.md

```diff
@@ -163,10 +170,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="$OVN_CENTRAL_IP:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
```

Comment:
The difference is "-e" (for editable, I think?).  Without "-e" installs like you would normally expect.